### PR TITLE
Create .entangle_profile

### PR DESCRIPTION
**Pull request by infiniti8**

### CODE REVIEW

File: profiles/.entangle_profile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# Entangle Wallet Name (`entangled keys list`)
+# Example: DELEGATOR_NAME="EZStaking"
+DELEGATOR_NAME=""
+
+# Entangle Wallet Address (`entangled keys show EntangleWalletName -a`)
+# Example: DELEGATOR="ethm19j4pje5sljgta7t79c0xh6uduc8u7ra0l45d6j"
+DELEGATOR=""
+
+# Entangle Validator Address
+# Example: VALIDATOR="evmosvaloper19edzsptn6t0v7clgyt7v0xs4t6ucw5nf5un6gh"
```
Comment (zakarialounes) [inline profiles/.entangle_profile:20]: please update for entangle

File: profiles/.entangle_profile [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+##########################################################
+
+# The IP of your Entangle Node
+NODE_IP="127.0.0.1"
+
+# The port of your Entangle Node
+NODE_PORT="26657"
+
+# The home of your Entangle Node
+# Example: /root/.entangle
```
Comment (zakarialounes) [inline profiles/.entangle_profile:42]: Please remove this line

File: profiles/.entangle_profile [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+NODE_HOME=""
```
Comment (zakarialounes) [inline profiles/.entangle_profile:43]: NODE_HOME="$HOME/.entangle"

File: profiles/.entangle_profile [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+# The path to your Entangle binary
+# To get the full path to your Entangle binary you can use `whereis entangled` and replace the below value by.
+BINARY=""
```
Comment (zakarialounes) [inline profiles/.entangle_profile:47]: set a value like `BINARY="$HOME/go/bin/bitsongcli"`

**Review by zakarialounes — CHANGES_REQUESTED** [review]

File: profiles/.entangle_profile [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+##################
+# DO NOT MODIFY! #
+##################
+
+# Depending on Cosmos SDK version
+# - ACCOUNT     $binary query account
+# - BANK        $binary query bank balances
+BALANCES_FROM="BANK"
+
+#SDETAILS="--chain-id ${CHAIN_ID} --node tcp://${NODE_IP}:${NODE_PORT} --home ${NODE_HOME}"
```
Comment (zakarialounes) [inline profiles/.entangle_profile:70]: remove this line

File: profiles/.entangle_profile [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+SDETAILS="--chain-id ${CHAIN_ID} --node tcp://${NODE_IP}:${NODE_PORT} --home ${NODE_HOME}"
+#DETAILS="${SDETAILS} --keyring-backend ${KEYRING_BACKEND}"
```
Comment (zakarialounes) [inline profiles/.entangle_profile:72]: uncomment this line

File: profiles/.entangle_profile [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+DETAILS="${SDETAILS}"
```
Comment (zakarialounes) [inline profiles/.entangle_profile:73]: remove this line

**Review by zakarialounes — CHANGES_REQUESTED** [review]

**Comment by infiniti8** [comment]
done, ty

**Pull request closed without merging**